contrib/!source-control/github/packages.el

```diff
@@ -86,3 +88,36 @@
         "gfC" 'spacemacs/git-link-commit-copy-url-only)
       ;; default is to open the generated link
       (setq git-link-open-in-browser t))))
+
+;; magit-gh-pulls has to be loaded via a pre-init hook because the source code
+;; makes assumptions about the status of the magit-mode keymaps that are
+;; incompatible with the spacemacs' evilification feature.
+;; To avoid errors, magit-gh-pulls must be loaded after magit, but before magit
+;; is configured by spacemacs. 
+
+(defun github/init-magit-gh-pulls ())
+
+(defun github/pre-init-magit ()
+  (spacemacs|use-package-add-hook magit
+    :pre-config
+    (progn
+      (use-package magit-gh-pulls
+        ;; allow users to exclude magit-gh-pulls as they normally would
+        :if (configuration-layer/package-usedp 'magit-gh-pulls)
```

Comment:
@syl20bnr Would it make sense for me to add an additional case so that the code only runs if magit-gh-pulls is used, and the github layer is the owner? As of right now, excluding magit-gh-pulls should work ok, but if another layer introduces its own init, both will run.